src/adera/Machines/RCSController.cpp

```diff
@@ -1,6 +1,6 @@
 /**
  * Open Space Program
- * Copyright � 2019-2020 Open Space Program Project
+ * Copyright © 2019-2020 Open Space Program Project
```

Comment:
My IDE had been bothering me about the encoding of this unicode copyright symbol. Also noting bumping the copyright to 2021.